moped-api/users/helpers.py

```diff
@@ -283,9 +273,7 @@ def cognito_user_exists(user_list_response: dict, user_email: str) -> tuple:
     """
     # Retrieves the full list of users, but removes any azuread emails
     user_list_filtered = list(
-        filter(
-            lambda user: "azuread_" not in user["Username"], user_list_response["Users"]
-        )
+        filter(lambda user: "azuread_" not in user["Username"], user_list_response)
```

Comment:
user_list_response is now a list, instead of a dict. 

everything above this was just linting